<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment> All rights reserved. </comment>
 <class>CQMergingData</class>
 <widget class="QDialog" name="CQMergingData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Merging Data</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Replace:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>with:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTreeWidget" name="mpTree2">
       <attribute name="headerVisible">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">model elements</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>.</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTreeWidget" name="mpTree1">
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="headerVisible">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">model elements</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>.</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnCancel">
       <property name="text">
        <string>Done</string>
       </property>
       <property name="shortcut">
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnMerge">
       <property name="text">
        <string>&amp;Merge</string>
       </property>
       <property name="shortcut">
        <string>Alt+O</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>mpBtnMerge</tabstop>
 </tabstops>
 <includes>
  <include location="global">string</include>
  <include location="global">map</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>mpBtnMerge</sender>
   <signal>clicked()</signal>
   <receiver>CQMergingData</receiver>
   <slot>slotBtnMerge()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnCancel</sender>
   <signal>clicked()</signal>
   <receiver>CQMergingData</receiver>
   <slot>slotBtnCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
